en/flight_controller/cuav_v5_plus.md

```diff
@@ -109,6 +109,47 @@ It does not have an i/o debug interface.
 Any multicopter / airplane / rover or boat that can be controlled with normal RC servos or Futaba S-Bus servos.
 The complete set of supported configurations can be seen in the [Airframes Reference](../airframes/airframe_reference.md).
 
+## Known Issues
+
+#### GPS not compatible with other devices (Critical)
+
+The *Neo v2.0 GPS* recommended for use with *CUAV V5+* and *CUAV V5 nano* is not fully compatible with other Pixhawk flight controllers (specifically, the buzzer part is not compatible and there may be issues with the safety switch).
+The GPS will not work with other flight controllers, and is the only GPS unit that can be used with the *CUAV V5+* and *CUAV V5 nano*.
+<!-- 5+/90/V5+ 20190523 RC01 -->
+
+- *Found:* Batch 01
+- *Fixed:* -
+
+#### Volt regulation varies greater than +/- 5%
+
+The 5 volt pins on all connectors will be lower when powered from USB than the Power Module (the pins will measure approximately 4.69V when only powered by USB, and 5.28 Volts when connected to the Power Module).
+
+> **Note** We recommend that when using USB with the *v5 Nano* you *also connect the power module* (to avoid under-powering any connected peripherals.
```

Comment:
@davids5 I changed as above, but this is the v5+ version rather than the nano. Is the behaviour exactly the same? ie are the voltages the same, so the only change I need to make is to "when using USB with the *v5+* ?

Reply:
It is on both See AL log.

Reply:
Perhaps I used the wrong word. We are telling them to test with a PM on the bench so we should also Add a warning to Remove Props before bench testing.